lib/src/container/ociwriter.rs

```diff
@@ -133,7 +133,22 @@ impl<'a> OciWriter<'a> {
 
     /// Add a layer to the top of the image stack.  The firsh pushed layer becomes the root.
     pub(crate) fn push_layer(&mut self, layer: Layer) {
-        self.layers.push(layer)
+        let v: Option<HashMap<String, String>> = None;
+        self.push_layer_annotated(layer, v);
+    }
+
+    /// Add a layer to the top of the image stack.  The firsh pushed layer becomes the root.
```

Comment:
`firsh` -> `first`

Ahh, this is copy/pasted. Seems like one of the two docstrings should differ to mention annotations handling.